src/__experimental__/components/date/date.spec.js

```diff
@@ -479,6 +480,35 @@ describe('Date', () => {
       });
     });
 
+    // This test is for a specific bug found in Storybook where the component can be focused,
+    // the knobs for readOnly and/or disabled can then be selected and the date component can then
+    // still be interacted with.
+    it('short circuits when either "disabled" and/or "readOnly"', () => {
```

Comment:
@DipperTheDan this test can be removed lines 486 -> 494

Reply:
Yep. Just realised that too. The purpose of the new test was to replace this.